awx/settings/defaults.py

```diff
@@ -1001,7 +1006,7 @@ def IS_TESTING(argv=None):
             'level': 'INFO',  # very verbose debug-level logs
         },
         'awx.analytics': {
-            'handlers': ['task_system', 'external_logger'],
```

Comment:
Specific revert of a commit in #7709, see https://github.com/ansible/awx/pull/7709#discussion_r489877104